app/pages/login.coffee

```diff
@@ -10,6 +10,8 @@ $(document).ready( tc (event) ->
         @redirectPage = 'index'
       pimatic.socket.io.disconnect()
       jQuery.mobile.changePage '#login-page', transition: 'flip'
+      $("#login-page").bind 'pageshow', ->
+        $('#loginForm  #firstName').focus()
```

Comment:
Looks like a good idea. @mwittig Could you change it so that it is bound to the document via on instead (like the pagebeforeshow handler some lines below)?